lib/metrics/metrics.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
+// Licensed under the Apache License, Version 2.0. See LICENSE
+
+export default {
+  TTFCP: 'first-contentful-paint',
+  TTFMP: 'first-meaningful-paint',
+  TTF_CPU_IDLE: 'first-cpu-idle',
+  TTCI: 'interactive',
+  PSI: 'speed-index',
```

Comment:
PSI -> SI everywhere. we ditched perceptual speed index and now just calculate speed index.